salt/modules/elasticsearch.py

```diff
@@ -264,6 +264,59 @@ def cluster_stats(nodes=None, hosts=None, profile=None):
         raise CommandExecutionError("Cannot retrieve cluster stats, server returned code {0} with message {1}".format(e.status_code, e.error))
 
 
+def cluster_get_settings(flat_settings=False, include_defaults=False, hosts=None, profile=None):
+    '''
+    .. versionadded:: Sodium
```

Comment:
```suggestion
    .. versionadded:: Neon
```
this will be in neon

Reply:
done